change/@fluentui-react-spinbutton-cffa2c98-0617-4b61-ac17-e7b6d96dccb1.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 256 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26867&commitId=58091125b32c22d3c62f3c18eb223d1f693c7f4b&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1592 screenshots unchanged, 0 screenshots with different dimensions and 256 screenshots with visible difference.


  <details><summary><b>unknown</b> 256 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Field.SpinButton-error.default.chromium.png|12| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.focused.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverDecrement.chromium.png|13| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverIncrement.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.hoverInput.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownDecrement.chromium.png|16| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.mouseDownIncrement.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker - Dark Mode.rest.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.focused.chromium.png|20| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverDecrement.chromium.png|13| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverIncrement.chromium.png|20| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.hoverInput.chromium.png|20| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownDecrement.chromium.png|13| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.mouseDownIncrement.chromium.png|20| Changed|
|SpinButton Converged.Appearance- filled-darker - High Contrast.rest.chromium.png|20| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.focused.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverDecrement.chromium.png|13| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverIncrement.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.hoverInput.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownDecrement.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.mouseDownIncrement.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker - RTL.rest.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker.focused.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker.hoverDecrement.chromium.png|13| Changed|
|SpinButton Converged.Appearance- filled-darker.hoverIncrement.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker.hoverInput.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker.mouseDownDecrement.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker.mouseDownIncrement.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-darker.rest.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.focused.chromium.png|10| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverDecrement.chromium.png|11| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverIncrement.chromium.png|10| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.hoverInput.chromium.png|10| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownDecrement.chromium.png|14| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.mouseDownIncrement.chromium.png|10| Changed|
|SpinButton Converged.Appearance- filled-lighter - Dark Mode.rest.chromium.png|10| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.focused.chromium.png|20| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverDecrement.chromium.png|13| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverIncrement.chromium.png|20| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.hoverInput.chromium.png|20| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownDecrement.chromium.png|13| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.mouseDownIncrement.chromium.png|20| Changed|
|SpinButton Converged.Appearance- filled-lighter - High Contrast.rest.chromium.png|20| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.focused.chromium.png|12| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverDecrement.chromium.png|14| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverIncrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.hoverInput.chromium.png|12| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownDecrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.mouseDownIncrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- filled-lighter - RTL.rest.chromium.png|12| Changed|
|SpinButton Converged.Appearance- filled-lighter.focused.chromium.png|12| Changed|
|SpinButton Converged.Appearance- filled-lighter.hoverDecrement.chromium.png|14| Changed|
|SpinButton Converged.Appearance- filled-lighter.hoverIncrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- filled-lighter.hoverInput.chromium.png|12| Changed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownDecrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- filled-lighter.mouseDownIncrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- filled-lighter.rest.chromium.png|12| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.focused.chromium.png|10| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverDecrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverIncrement.chromium.png|10| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.hoverInput.chromium.png|10| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownDecrement.chromium.png|13| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.mouseDownIncrement.chromium.png|10| Changed|
|SpinButton Converged.Appearance- outline (default) - Dark Mode.rest.chromium.png|10| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.focused.chromium.png|20| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverDecrement.chromium.png|13| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverIncrement.chromium.png|20| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.hoverInput.chromium.png|20| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownDecrement.chromium.png|13| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.mouseDownIncrement.chromium.png|20| Changed|
|SpinButton Converged.Appearance- outline (default) - High Contrast.rest.chromium.png|20| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.focused.chromium.png|12| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverDecrement.chromium.png|14| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverIncrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.hoverInput.chromium.png|12| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownDecrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.mouseDownIncrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- outline (default) - RTL.rest.chromium.png|12| Changed|
|SpinButton Converged.Appearance- outline (default).focused.chromium.png|12| Changed|
|SpinButton Converged.Appearance- outline (default).hoverDecrement.chromium.png|14| Changed|
|SpinButton Converged.Appearance- outline (default).hoverIncrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- outline (default).hoverInput.chromium.png|12| Changed|
|SpinButton Converged.Appearance- outline (default).mouseDownDecrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- outline (default).mouseDownIncrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- outline (default).rest.chromium.png|12| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.focused.chromium.png|10| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverDecrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverIncrement.chromium.png|10| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.hoverInput.chromium.png|10| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownDecrement.chromium.png|13| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.mouseDownIncrement.chromium.png|10| Changed|
|SpinButton Converged.Appearance- underline - Dark Mode.rest.chromium.png|10| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.focused.chromium.png|20| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverDecrement.chromium.png|13| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverIncrement.chromium.png|20| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.hoverInput.chromium.png|20| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownDecrement.chromium.png|13| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.mouseDownIncrement.chromium.png|20| Changed|
|SpinButton Converged.Appearance- underline - High Contrast.rest.chromium.png|20| Changed|
|SpinButton Converged.Appearance- underline - RTL.focused.chromium.png|12| Changed|
|SpinButton Converged.Appearance- underline - RTL.hoverDecrement.chromium.png|14| Changed|
|SpinButton Converged.Appearance- underline - RTL.hoverIncrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- underline - RTL.hoverInput.chromium.png|12| Changed|
|SpinButton Converged.Appearance- underline - RTL.mouseDownDecrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- underline - RTL.mouseDownIncrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- underline - RTL.rest.chromium.png|12| Changed|
|SpinButton Converged.Appearance- underline.focused.chromium.png|12| Changed|
|SpinButton Converged.Appearance- underline.hoverDecrement.chromium.png|14| Changed|
|SpinButton Converged.Appearance- underline.hoverIncrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- underline.hoverInput.chromium.png|12| Changed|
|SpinButton Converged.Appearance- underline.mouseDownDecrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- underline.mouseDownIncrement.chromium.png|12| Changed|
|SpinButton Converged.Appearance- underline.rest.chromium.png|12| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.focused.chromium.png|10| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverDecrement.chromium.png|12| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverIncrement.chromium.png|10| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.hoverInput.chromium.png|10| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownDecrement.chromium.png|13| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.mouseDownIncrement.chromium.png|10| Changed|
|SpinButton Converged.At Max Bound - Dark Mode.rest.chromium.png|10| Changed|
|SpinButton Converged.At Max Bound - High Contrast.focused.chromium.png|20| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverDecrement.chromium.png|13| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverIncrement.chromium.png|20| Changed|
|SpinButton Converged.At Max Bound - High Contrast.hoverInput.chromium.png|20| Changed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownDecrement.chromium.png|13| Changed|
|SpinButton Converged.At Max Bound - High Contrast.mouseDownIncrement.chromium.png|20| Changed|
|SpinButton Converged.At Max Bound - High Contrast.rest.chromium.png|20| Changed|
|SpinButton Converged.At Max Bound - RTL.focused.chromium.png|12| Changed|
|SpinButton Converged.At Max Bound - RTL.hoverDecrement.chromium.png|14| Changed|
|SpinButton Converged.At Max Bound - RTL.hoverIncrement.chromium.png|12| Changed|
|SpinButton Converged.At Max Bound - RTL.hoverInput.chromium.png|12| Changed|
|SpinButton Converged.At Max Bound - RTL.mouseDownDecrement.chromium.png|12| Changed|
|SpinButton Converged.At Max Bound - RTL.mouseDownIncrement.chromium.png|12| Changed|
|SpinButton Converged.At Max Bound - RTL.rest.chromium.png|12| Changed|
|SpinButton Converged.At Max Bound.focused.chromium.png|12| Changed|
|SpinButton Converged.At Max Bound.hoverDecrement.chromium.png|14| Changed|
|SpinButton Converged.At Max Bound.hoverIncrement.chromium.png|12| Changed|
|SpinButton Converged.At Max Bound.hoverInput.chromium.png|12| Changed|
|SpinButton Converged.At Max Bound.mouseDownDecrement.chromium.png|12| Changed|
|SpinButton Converged.At Max Bound.mouseDownIncrement.chromium.png|12| Changed|
|SpinButton Converged.At Max Bound.rest.chromium.png|12| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.focused.chromium.png|4| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverDecrement.chromium.png|4| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverIncrement.chromium.png|4| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.hoverInput.chromium.png|4| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownDecrement.chromium.png|4| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.mouseDownIncrement.chromium.png|4| Changed|
|SpinButton Converged.At Min Bound - Dark Mode.rest.chromium.png|4| Changed|
|SpinButton Converged.At Min Bound - High Contrast.focused.chromium.png|13| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverDecrement.chromium.png|13| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverIncrement.chromium.png|13| Changed|
|SpinButton Converged.At Min Bound - High Contrast.hoverInput.chromium.png|13| Changed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownDecrement.chromium.png|13| Changed|
|SpinButton Converged.At Min Bound - High Contrast.mouseDownIncrement.chromium.png|13| Changed|
|SpinButton Converged.At Min Bound - High Contrast.rest.chromium.png|13| Changed|
|SpinButton Converged.At Min Bound - RTL.focused.chromium.png|5| Changed|
|SpinButton Converged.At Min Bound - RTL.hoverDecrement.chromium.png|5| Changed|
|SpinButton Converged.At Min Bound - RTL.hoverIncrement.chromium.png|5| Changed|
|SpinButton Converged.At Min Bound - RTL.hoverInput.chromium.png|5| Changed|
|SpinButton Converged.At Min Bound - RTL.mouseDownDecrement.chromium.png|5| Changed|
|SpinButton Converged.At Min Bound - RTL.mouseDownIncrement.chromium.png|5| Changed|
|SpinButton Converged.At Min Bound - RTL.rest.chromium.png|5| Changed|
|SpinButton Converged.At Min Bound.focused.chromium.png|5| Changed|
|SpinButton Converged.At Min Bound.hoverDecrement.chromium.png|5| Changed|
|SpinButton Converged.At Min Bound.hoverIncrement.chromium.png|5| Changed|
|SpinButton Converged.At Min Bound.hoverInput.chromium.png|5| Changed|
|SpinButton Converged.At Min Bound.mouseDownDecrement.chromium.png|5| Changed|
|SpinButton Converged.At Min Bound.mouseDownIncrement.chromium.png|5| Changed|
|SpinButton Converged.At Min Bound.rest.chromium.png|5| Changed|
|SpinButton Converged.Disabled - Dark Mode.focused.chromium.png|4| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverDecrement.chromium.png|4| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverIncrement.chromium.png|4| Changed|
|SpinButton Converged.Disabled - Dark Mode.hoverInput.chromium.png|4| Changed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownDecrement.chromium.png|4| Changed|
|SpinButton Converged.Disabled - Dark Mode.mouseDownIncrement.chromium.png|4| Changed|
|SpinButton Converged.Disabled - Dark Mode.rest.chromium.png|4| Changed|
|SpinButton Converged.Disabled - High Contrast.focused.chromium.png|13| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverDecrement.chromium.png|13| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverIncrement.chromium.png|13| Changed|
|SpinButton Converged.Disabled - High Contrast.hoverInput.chromium.png|13| Changed|
|SpinButton Converged.Disabled - High Contrast.mouseDownDecrement.chromium.png|13| Changed|
|SpinButton Converged.Disabled - High Contrast.mouseDownIncrement.chromium.png|13| Changed|
|SpinButton Converged.Disabled - High Contrast.rest.chromium.png|13| Changed|
|SpinButton Converged.Disabled - RTL.focused.chromium.png|5| Changed|
|SpinButton Converged.Disabled - RTL.hoverDecrement.chromium.png|5| Changed|
|SpinButton Converged.Disabled - RTL.hoverIncrement.chromium.png|5| Changed|
|SpinButton Converged.Disabled - RTL.hoverInput.chromium.png|5| Changed|
|SpinButton Converged.Disabled - RTL.mouseDownDecrement.chromium.png|5| Changed|
|SpinButton Converged.Disabled - RTL.mouseDownIncrement.chromium.png|5| Changed|
|SpinButton Converged.Disabled - RTL.rest.chromium.png|5| Changed|
|SpinButton Converged.Disabled.focused.chromium.png|5| Changed|
|SpinButton Converged.Disabled.hoverDecrement.chromium.png|5| Changed|
|SpinButton Converged.Disabled.hoverIncrement.chromium.png|5| Changed|
|SpinButton Converged.Disabled.hoverInput.chromium.png|5| Changed|
|SpinButton Converged.Disabled.mouseDownDecrement.chromium.png|5| Changed|
|SpinButton Converged.Disabled.mouseDownIncrement.chromium.png|5| Changed|
|SpinButton Converged.Disabled.rest.chromium.png|5| Changed|
|SpinButton Converged.Display Value - Dark Mode.focused.chromium.png|10| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverDecrement.chromium.png|12| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverIncrement.chromium.png|10| Changed|
|SpinButton Converged.Display Value - Dark Mode.hoverInput.chromium.png|10| Changed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownDecrement.chromium.png|13| Changed|
|SpinButton Converged.Display Value - Dark Mode.mouseDownIncrement.chromium.png|10| Changed|
|SpinButton Converged.Display Value - Dark Mode.rest.chromium.png|10| Changed|
|SpinButton Converged.Display Value - High Contrast.focused.chromium.png|20| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverDecrement.chromium.png|13| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverIncrement.chromium.png|20| Changed|
|SpinButton Converged.Display Value - High Contrast.hoverInput.chromium.png|20| Changed|
|SpinButton Converged.Display Value - High Contrast.mouseDownDecrement.chromium.png|13| Changed|
|SpinButton Converged.Display Value - High Contrast.mouseDownIncrement.chromium.png|20| Changed|
|SpinButton Converged.Display Value - High Contrast.rest.chromium.png|20| Changed|
|SpinButton Converged.Display Value - RTL.focused.chromium.png|12| Changed|
|SpinButton Converged.Display Value - RTL.hoverDecrement.chromium.png|14| Changed|
|SpinButton Converged.Display Value - RTL.hoverIncrement.chromium.png|12| Changed|
|SpinButton Converged.Display Value - RTL.hoverInput.chromium.png|12| Changed|
|SpinButton Converged.Display Value - RTL.mouseDownDecrement.chromium.png|12| Changed|
|SpinButton Converged.Display Value - RTL.mouseDownIncrement.chromium.png|12| Changed|
|SpinButton Converged.Display Value - RTL.rest.chromium.png|12| Changed|
|SpinButton Converged.Display Value.focused.chromium.png|12| Changed|
|SpinButton Converged.Display Value.hoverDecrement.chromium.png|14| Changed|
|SpinButton Converged.Display Value.hoverIncrement.chromium.png|12| Changed|
|SpinButton Converged.Display Value.hoverInput.chromium.png|12| Changed|
|SpinButton Converged.Display Value.mouseDownDecrement.chromium.png|12| Changed|
|SpinButton Converged.Display Value.mouseDownIncrement.chromium.png|12| Changed|
|SpinButton Converged.Display Value.rest.chromium.png|12| Changed|
|SpinButton Converged.Invalid- filled-darker.focused.chromium.png|11| Changed|
|SpinButton Converged.Invalid- filled-darker.hoverDecrement.chromium.png|13| Changed|
|SpinButton Converged.Invalid- filled-darker.hoverIncrement.chromium.png|11| Changed|
|SpinButton Converged.Invalid- filled-darker.hoverInput.chromium.png|11| Changed|
|SpinButton Converged.Invalid- filled-darker.mouseDownDecrement.chromium.png|11| Changed|
|SpinButton Converged.Invalid- filled-darker.mouseDownIncrement.chromium.png|11| Changed|
|SpinButton Converged.Invalid- filled-darker.rest.chromium.png|11| Changed|
|SpinButton Converged.Invalid- filled-lighter.focused.chromium.png|12| Changed|
|SpinButton Converged.Invalid- filled-lighter.hoverDecrement.chromium.png|14| Changed|
|SpinButton Converged.Invalid- filled-lighter.hoverIncrement.chromium.png|12| Changed|
|SpinButton Converged.Invalid- filled-lighter.hoverInput.chromium.png|12| Changed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownDecrement.chromium.png|12| Changed|
|SpinButton Converged.Invalid- filled-lighter.mouseDownIncrement.chromium.png|12| Changed|
|SpinButton Converged.Invalid- filled-lighter.rest.chromium.png|12| Changed|
|SpinButton Converged.Invalid- outline.focused.chromium.png|12| Changed|
|SpinButton Converged.Invalid- outline.hoverDecrement.chromium.png|14| Changed|
|SpinButton Converged.Invalid- outline.hoverIncrement.chromium.png|12| Changed|
|SpinButton Converged.Invalid- outline.hoverInput.chromium.png|12| Changed|
|SpinButton Converged.Invalid- outline.mouseDownDecrement.chromium.png|12| Changed|
|SpinButton Converged.Invalid- outline.mouseDownIncrement.chromium.png|12| Changed|
|SpinButton Converged.Invalid- outline.rest.chromium.png|12| Changed|
|SpinButton Converged.Invalid- underline.focused.chromium.png|12| Changed|
|SpinButton Converged.Invalid- underline.hoverDecrement.chromium.png|14| Changed|
|SpinButton Converged.Invalid- underline.hoverIncrement.chromium.png|12| Changed|
|SpinButton Converged.Invalid- underline.hoverInput.chromium.png|12| Changed|
|SpinButton Converged.Invalid- underline.mouseDownDecrement.chromium.png|12| Changed|
|SpinButton Converged.Invalid- underline.mouseDownIncrement.chromium.png|12| Changed|
|SpinButton Converged.Invalid- underline.rest.chromium.png|12| Changed|
|SpinButton Converged.Size- medium (default) - RTL.default.chromium.png|12| Changed|
|SpinButton Converged.Size- medium (default).default.chromium.png|12| Changed|
|SpinButton Converged.With appearance override.default.chromium.png|23| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>